### PR TITLE
Add ability to toggle IR targets via ROS topic

### PR DESCRIPTION
**Pull request by iche033**
I decided to use ROS topic for toggling the IR targets instead of a ROS param server to keep things simple.

To test, launch the demo world as usual:

~~~
roslaunch ir_beacon ir_beacon.launch
~~~

this reads the 4 default targets from SDF.

you can then change the list of targets by publishing to the `/ir_targets` topic, e.g.

~~~
rostopic pub -1 /ir_targets ir_beacon/Targets "{targets:['ir_beacon::led_mount::IR_LED_01', 'ir_beacon::led_mount::IR_LED_04']}"
~~~

if you have `rqt image view` open, you should see that now there are only 2 IR targets being tracked instead of 4

### CODE REVIEW

File: ir_beacon/CMakeLists.txt [hunk 1]
```diff
@@ -66,10 +69,10 @@ find_package(catkin REQUIRED COMPONENTS
 # )
 
 ## Generate added messages and services with any dependencies listed here
-# generate_messages(
-#   DEPENDENCIES
-#   std_msgs  # Or other packages containing msgs
-# )
+ generate_messages(
+   DEPENDENCIES
+   std_msgs  # Or other packages containing msgs
```
Comment (tfoote) [inline ir_beacon/CMakeLists.txt]: Is std_msgs necessary? The message looks like it's only using primatives.

File: ir_beacon/plugins/IRSensorPlugin.cc [hunk 2]
```diff
@@ -223,10 +241,63 @@ void IRMaterialHandler::TagIRVisual(rendering::VisualPtr _vis)
...
+}
+
 
 /////////////////////////////////////////////////
 void IRMaterialHandler::PreRender()
 {
+  // remove existing targets
+  for (auto vis : this->targetVisuals)
+  {
+    this->RemoveTag(vis);
```
Comment (tfoote) [inline ir_beacon/plugins/IRSensorPlugin.cc]: Is this going to remove the tag at every render iteration to then be added back down below?

File: ir_beacon/CMakeLists.txt (hunk 1 above)
Comment (iche033) [reply ir_beacon/CMakeLists.txt]: true, removed dependency on std_msgs af4ae88

File: ir_beacon/plugins/IRSensorPlugin.cc (hunk 2 above)
Comment (iche033) [reply ir_beacon/plugins/IRSensorPlugin.cc]: good catch, I added a flag to make sure it only removes the visuals only when new targets are set. af4ae88

**Pull request merged**